cds_ils/importer/importer.py

```diff
@@ -346,7 +363,7 @@ def preview_delete_document(self, document):
         related_refs = set()
         for _, related_objects in document.relations.items():
             for obj in related_objects:
-                if not obj["record_metadata"].get("mode_of_issuance")\
+                if not obj["record_metadata"].get("mode_of_issuance") \
```

Comment:
is it ok to use `.get` here? can it be unset?

Reply:
there are other related record types which don't have the mode of issuance